Modix.sln

```diff
@@ -27,6 +27,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Modix", "Modix\Modix.csproj
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Modix.Services.Test", "Modix.Services.Test\Modix.Services.Test.csproj", "{D9BFC7C7-10B2-4D61-99EA-79805074CCD7}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Modix.Common", "Modix.Common\Modix.Common.csproj", "{575B80E6-56E5-4E9A-94EE-ADEB3A1A9FC0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Modix.Common.Test", "Modix.Common.Test\Modix.Common.Test.csproj", "{4F9BDC85-B8B2-4AC5-99BC-1F2F0CF80016}"
+EndProject
```

Comment:
Won't these project references cause errors on solution load, since this PR didn't add them and they aren't checked into the repo yet?

Reply:
Uhhh....sort of? I verified that the solution loads and all tests run before committing, but now that I look I can see that `Modix.Common` and `Modix.Common.Test` are in the Solution Explorer, but simply show as "Unavailable". It came out of a dumb merge conflict when I was rebasing to the latest master, I'll see if I can clean it up.